src/Sulu/Bundle/ContactBundle/Resources/views/Template/base.form.html.twig

```diff
@@ -38,7 +38,7 @@
                     <script type="text/template" id="address-tpl">
                         <div class="grid-col-12 floating m-top-0 address-component">
                             <~ var uid = _.uniqueId('address_'), first = true; ~>
-                            <~ var getAddressString=function(street,number,zip,city,country){var str=!!street?street:"",part=(zip+" "+city).trim();str+=!!str.length&&!!number?" "+number:number;str+=!!str.length&&!!part?", "+part:part;str+=!!str.length&&!!country?", "+country:country;return str}; ~>
+                            <~ var getAddressString=function(title,street,number,zip,city,country){var str=(!!title?title:""),part=(zip+" "+city).trim();str+=(!!street?(!!str.length?", ":"") + street:"");str+=!!str.length&&!!number?" "+number:number;str+=!!str.length&&!!part?", "+part:part;str+=!!str.length&&!!country?", "+country:country;return str}; ~>
```

Comment:
its not to easy to extract the logic because we cannot use `!!str.length` and then you have a big statement like: `!!street && !!number && !!country ...` and thats also weird ...